packages/js/src/core/user-info.ts

```diff
@@ -0,0 +1,18 @@
+import { Requester } from '../types';
+
+export type UserInfoResponse = {
+  sub: string;
+  name?: string;
+  username?: string;
+  avatar?: string;
+  role_names?: string[];
+};
+
+export const fetchUserInfo = async (
+  userInfoEndpoint: string,
+  accessToken: string,
+  requester: Requester
+): Promise<UserInfoResponse> =>
+  requester<UserInfoResponse>(userInfoEndpoint, {
+    headers: { Authorization: `Bearer ${accessToken}` },
```

Comment:
In node sdk, we mutate the "headers", would you mind checking if there is a conflict issue?

Reply:
for appSecret

Reply:
Hmmm, I think you're right. Should move this to browser SDK

Reply:
But seems OK for now, this endpoint does not require `appSecret` to work?

Reply:
No, it does not require appSecret. Yeah, I thought about this couple days ago but forgot to update the comment. So I guess it's fine for now